resources/definitions/fdmprinter.def.json

```diff
@@ -3290,6 +3290,16 @@
             "description": "Adhesion",
             "children":
             {
+                "prime_poop_enable":
+                {
+                    "label": "Enable Prime Blob",
+                    "description": "Whether to prime the filament with a blob before printing.",
```

Comment:
Please explain what benefit the user has by changing this setting.

The description should explain to a user why and when he should change the setting.

It should explain the benefit of turning it off and the benefit of turning it off.